Sources/Purchasing/Configuration.swift

```diff
@@ -376,11 +376,19 @@ extension Configuration {
 
     enum APIKeyValidationResult {
         case validApplePlatform
+        case testStore
         case otherPlatforms
         case legacy
     }
 
     static func validate(apiKey: String) -> APIKeyValidationResult {
+        #if TEST_STORE
+        if apiKey.hasPrefix(testStoreKeyPrefix) {
+            // Test Store key format: "test_CtDdmbdWBySmqJeeQUTyrNxETUVkajsJ"
+            return .testStore
+        }
+        #endif // TEST_STORE
```

Comment:
Having this under the `TEST_STORE` flag is enough, since the `APIKeyValidationResult` enum is an internal API